snow/consensus/snowman/topological.go

```diff
@@ -301,23 +321,31 @@ func (ts *Topological) RecordPoll(ctx context.Context, voteBag bag.Bag[ids.ID])
 		return nil
 	}
 
-	// Runtime = |live set| ; Space = Constant
+	// Runtime = 2 * |live set| ; Space = Constant
```

Comment:
irrelevant comment but sometimes i feel like the comments in the code are overly verbose haha...

Reply:
idk I feel like for a hot path where we care about the performance a lot it's nice